buildings/makefile

```diff
@@ -0,0 +1,101 @@
+#/***************************************************************************
+# qgis-building-plugin
+#
+# qgis-buildings-plugin
+#							 -------------------
+#		begin				: 2018-03-26
+#		git sha				: $Format:%H$
+#		copyright			: (C) 2018 by LINZ
+#		email				:
+# ***************************************************************************/
+
+
+PLUGINNAME = buildings
+
+AF = buildings/gui/i18n/af.ts
+
+deploy:
```

Comment:
This function doesn't work as it is trying to copy the buildings folder but we are now already in the buildings folder. We could instead move `deploy`, `set_up_test_db` and `PLUGINNAME` (and maybe `dclean`, `derase` and `zip` if you think they are useful) to the makefile in nz-buildings. The other functions in this file are no longer used, so we can remove them and this file.